csvqb/csvqb/tests/behaviour/qbwriter.feature

```diff
@@ -51,9 +51,60 @@ Feature: Test outputting CSV-Ws with Qb flavouring.
     Given a multi-measure QbCube named "Duplicate Qube"
     When the cube is serialised to CSV-W
     Then csvlint validation of "duplicate-qube.csv-metadata.json" should succeed
-    # todo: complete me in Issue #65
+  # todo: complete me in Issue #65
 
   Scenario: A multi-measure QbCube with duplicate rows should fail validation
     Given a multi-measure QbCube named "Duplicate Qube" with duplicate rows
     When the cube is serialised to CSV-W
-    Then csvlint validation of "duplicate-qube.csv-metadata.json" should fail with "duplicate_key"
\ No newline at end of file
+    Then csvlint validation of "duplicate-qube.csv-metadata.json" should fail with "duplicate_key"
+
+  Scenario: A QbCube with string literal attributes should validate successfully
+    Given a single-measure QbCube named "Qube with string literals" with a "string" attribute
+    When the cube is serialised to CSV-W
+    Then csvlint validation of "qube-with-string-literals.csv-metadata.json" should succeed
+    And csv2rdf on all CSV-Ws should succeed
+    And turtle should be written to "output.ttl"
+    And the RDF should contain
+      """
+      <file:/tmp/qube-with-string-literals.csv#obs/uss-cerritos> <file:/tmp/qube-with-string-literals.csv#attribute/first-captain>
+      "William Riker".
```

Comment:
What did you decide to do wrt language tags?

Reply:
They're a separate ticket. #131